en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -15,9 +15,9 @@ We'll go through each of these in detail in the following sections.
 
 | Main interface | Function |
 | :--- | :--- |
-| POWER1 | Connect power module; power input & AD voltage and current detection. |
-| POWER2 | Connect i2c smart battery. |
-| TF CARD | Insert SD card for log storage|
+| Power1 | Connect power module; power input & AD voltage and current detection. |
```

Comment:
FYI, the text is changed here because this is how they are labeled on the silk screen.

Reply:
Great

Reply:
@hamishwillee Change to 
| Power1 | Connect power module; power input with **ANALOG** voltage and current detection. 
Do not use a Digital PM on this connector!